src/plugins/discover/public/application/apps/main/components/sidebar/discover_sidebar.tsx

```diff
@@ -265,7 +267,7 @@ export function DiscoverSidebar({
 
   const filterChanged = useMemo(() => isEqual(fieldFilter, getDefaultFieldFilter()), [fieldFilter]);
 
-  if (!selectedIndexPattern || !fields) {
+  if (!selectedIndexPattern) {
```

Comment:
why removing this check? on the first render, fields can still be null

Reply:
catching this later on, else sidebar is not rendered until data is arriving